docs/build/program/apis/robot.md

```diff
@@ -885,4 +885,28 @@ For more information, see the [Typescript SDK Docs](https://ts.viam.dev/classes/
 {{% /tab %}}
 {{< /tabs >}}
 
+### GetCloudMetadata
+
+Returns app-related information about the robot.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None.
+
+**Returns:**
+
+- None.
```

Comment:
@maximpertsov I don't feel like this is correct but there's no info provided in the python sdk docs. I'm a bit confused.

Reply:
yeah it's wrong for sure, I'll make a follow-up to fix

Reply:
https://github.com/viamrobotics/viam-python-sdk/pull/549